app/styles/_story.scss

```diff
@@ -292,7 +292,13 @@ article > section a:hover {
   .embedded-image {
     display: block;
     position: relative;
-    max-width: 1440px;
```

Comment:
I'm curious, do you think this was not working before as a max-width?

Reply:
Without an `!important` it gets overridden by the inline style inserted by tinymce, which originates in publisher: https://github.com/nypublicradio/publisher/blob/master/puppy/media/js/tinymce_customplugins/puppyimage/plugin.js#L53